concent-builder/unit-tests.yml

```diff
@@ -0,0 +1,27 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: "{{ shared_user }}"
+      block:
+        - name: Configure kubectl to operate on the right cluster
+          command: gcloud container clusters get-credentials \
+              "{{ gke.cluster }}"                            \
+              --project "{{ gke.project }}"                  \
+              --zone    "{{ gke.zone }}"
+
+        - name:  Run postgres container for django unit tests
+          shell: docker ps -q --filter "name=postgres" | grep -q .               \
+              || docker run --rm -d --name postgres concent-postgresql:latest
+
+        - name:    Run concent-api container and django unit tests
+          command: docker run --rm                                                                                      \
+              -v {{ build_dir }}/concent-deployment/kubernetes/build/run-unit-tests.sh:/usr/local/bin/run-unit-tests.sh \
+              --link postgres concent-concent-api:latest /usr/local/bin/run-unit-tests.sh
+
+        - name:    Stop postgres container after tests were completed
+          command: docker stop postgres
```

Comment:
I know, but shell command check if container exist, so that's not a problem. When playbook's fail, we won't run the next condition. I think it's not necessary to stop the postgres container when some test fails in concent-api cotainer and then again run postgres with the same configuration.

Reply:
But if you want Postgres to run constantly then it's not the way to do this. You should just start the service in the `configure` playbook.

We want to either start and stop PostgreSQL for each test or keep it running constantly. Your solution is something in between.

Reply:
Done.